St_John's_College_MS_34.xml

```diff
@@ -0,0 +1,151 @@
+                     <incipit type="prologue" xml:lang="la">Imperatorie maiestatis est in palatio tres habere mansiones auditorium uel consistorium</incipit>
+                     <incipit type="text" xml:lang="la">In principio erat uerbum et uerbum erat principium in quo et per quod</incipit>
+                     <explicit xml:lang="la">in loco magno honorabili scilicet in cathacumbis.</explicit>
+                     <author key="person_40175167">Peter Comestor</author>
+                     <title key="work_3495">Historia scholastica</title>
+                     <note><bibl>(Stegmüller, RB 6345–65 [4:280–90]), ed. PL 198:1053–1722.</bibl> The final column is blank. A considerable amount of marginal addition of various sorts in a variety of hands (decreases substantially after Genesis).</note>
+                  </msItem>
+               </msContents>
+               <physDesc>
+                 <p><secFol>summitatem</secFol></p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>Vellum (<term type="disposition">HSOS</term>)</support>
+                        <extent>
+                           ii + 185 + iii (numbered iii-v)
+                           <dimensions type="leaf" unit="mm">
+                              <height quantity="335">335</height>
+                              <width quantity="212">212</width>
+                           </dimensions>
+                        </extent>
+                        <collation mainStructures="8"><formula>1–2(8), 3(8-1),4(8), 5(6), 6(8-3), 7-9(8), 10(10), 11-13(8), 14(8-1), 15-23(8), 24(4)</formula>
+                           <signatures>Signed on the final leaves of quires with a roman numeral. Although many signatures have been cut away, quires 6, 9–10, 12–16, and 23 have the expected signature in roman, ending xxiiius. on fol. 181v. Although the signature system is consistent, fols. 32–7 do not constitute a single quire and must reflect some difficulty in production. The six leaves have consecutive text (from <bibl>‘Exodus’, PL 198:1169–91</bibl>) in proper order, but currently appear to be a bifolium and a quire of four.</signatures>
+                           <catchwords>Catchwords under the gutter columns. Many catchwords have been cut away.</catchwords>
+                        </collation>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout>
+                           <dimensions type="column" unit="mm">
+                              <height quantity="205">205</height>
+                              <width min="53" max="55">53-55</width>
+                              </dimensions>
+                              <dimensions type="intercolumn" unit="mm">
+                                 <width quantity="11">11</width>
+                              </dimensions>
+                        </layout>
```

Comment:
Could you add some text to explain the dimensions? e.g. "Column dimensions ....; .... between columns." As it stands, only the numbers will appear when the xml is converted to html for web display. (This only applies to dimensions in the layout section.)